src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/MethodologySummary.test.tsx

```diff
@@ -168,99 +209,246 @@ describe('MethodologySummary', () => {
             onChangePublication={noop}
           />
         </MemoryRouter>,
       );
 
       expect(
-        screen.queryByText('Amendment', { selector: 'span' }),
+        screen.getByText('Amend methodology', { selector: 'button' }),
       ).toBeInTheDocument();
     });
-  });
 
-  describe('Has an external methodology', () => {
-    test('the external methodology link and buttons are shown', () => {
+    test('the amend methodology button is not shown if user does not have permission', () => {
       render(
         <MemoryRouter>
           <MethodologySummary
-            publication={testPublicationWithExternalMethodology}
+            publication={testPublicationWithMethodology}
             topicId={testTopicId}
             onChangePublication={noop}
           />
         </MemoryRouter>,
       );
 
       expect(
-        screen.queryByText('Ext methodolology title (external methodology)', {
-          selector: 'a',
-        }),
-      ).toBeInTheDocument();
+        screen.queryByText('Amend methodology', { selector: 'button' }),
+      ).not.toBeInTheDocument();
+    });
 
-      expect(screen.getByRole('button', { name: 'Edit' })).toBeInTheDocument();
+    test('shows the confirm modal when click the amend button', async () => {
+      render(
+        <MemoryRouter>
+          <MethodologySummary
+            publication={testPublicationWithMethodologyCanAmend}
+            topicId={testTopicId}
+            onChangePublication={noop}
+          />
+        </MemoryRouter>,
+      );
+
+      userEvent.click(
+        screen.getByText('Amend methodology', { selector: 'button' }),
+      );
+      await waitFor(() => {
+        expect(
+          screen.queryByText('Confirm you want to amend this live methodology'),
+        ).toBeInTheDocument();
+
+        expect(
+          screen.getByText('Confirm', { selector: 'button' }),
+        ).toBeInTheDocument();
+      });
+    });
+
+    test('calls the service to amend the methodology when the confirm button is clicked', async () => {
+      const history = createMemoryHistoryWithMockedPush();
+      const mockMethodology: BasicMethodology = {
+        amendment: true,
+        live: false,
+        id: '12345',
+        internalReleaseNote: 'this is the release note',
+        previousVersionId: 'lfkjdlfj',
+        published: '2021-06-08T09:04:17.9805585',
+        slug: 'meth-1',
+        status: 'Approved',
+        title: 'I am a methodology amendment',
+      };
+      methodologyService.createMethodologyAmendment.mockImplementation(() =>
+        Promise.resolve(mockMethodology),
+      );
+      render(
+        <Router history={history}>
+          <MethodologySummary
+            publication={testPublicationWithMethodologyCanAmend}
+            topicId={testTopicId}
+            onChangePublication={noop}
+          />
+        </Router>,
+      );
+      userEvent.click(
+        screen.getByText('Amend methodology', { selector: 'button' }),
```

Comment:
I'm guessing you can use getByRole here too

Reply:
This is the one I mentioned in Slack last week that I just can't get to work with getByRole for some reason :(